examples/notebooks/Watch_Warning_and_Advisory_Plotting.ipynb

```diff
@@ -0,0 +1,728 @@
+    "[4 Functions: get_title()](https://unidata.github.io/python-awips/examples/generated/Watch_Warning_and_Advisory_Plotting.html#functions-get-title)<br>  \n",
+    "[5 Initial Setup](https://unidata.github.io/python-awips/examples/generated/Watch_Warning_and_Advisory_Plotting.html#initial-setup)<br>  \n",
+    "&nbsp;&nbsp;&nbsp;&nbsp;[5.1 EDEX Connection](https://unidata.github.io/python-awips/examples/generated/Watch_Warning_and_Advisory_Plotting.html#edex-connection)<br>  \n",
+    "&nbsp;&nbsp;&nbsp;&nbsp;[5.2 Significance (Sig) Constraints](https://unidata.github.io/python-awips/examples/generated/Watch_Warning_and_Advisory_Plotting.html#significance-sig-constraints)<br>  \n",
+    "[6 Filter by Time](https://unidata.github.io/python-awips/examples/generated/Watch_Warning_and_Advisory_Plotting.html#filter-by-time)<br>  \n",
+    "[7 Use the Data!](https://unidata.github.io/python-awips/examples/generated/Watch_Warning_and_Advisory_Plotting.html#use-the-data)<br>  \n",
+    "&nbsp;&nbsp;&nbsp;&nbsp;[7.1 Get the Data](https://unidata.github.io/python-awips/examples/generated/Watch_Warning_and_Advisory_Plotting.html#get-the-data)<br>  \n",
+    "&nbsp;&nbsp;&nbsp;&nbsp;[7.2 Extract Phensigs, Geometries and Times](https://unidata.github.io/python-awips/examples/generated/Watch_Warning_and_Advisory_Plotting.html#extract-phensigs-geometries-and-times)<br>  \n",
+    "[8 Plot the Data!](https://unidata.github.io/python-awips/examples/generated/Watch_Warning_and_Advisory_Plotting.html#plot-the-data)<br>  \n",
+    "&nbsp;&nbsp;&nbsp;&nbsp;[8.1 Create State and Political Boundaries](https://unidata.github.io/python-awips/examples/generated/Watch_Warning_and_Advisory_Plotting.html#create-state-and-political-boundaries)<br>  \n",
+    "&nbsp;&nbsp;&nbsp;&nbsp;[8.2 Draw the Plot and Legend for WWAs](https://unidata.github.io/python-awips/examples/generated/Watch_Warning_and_Advisory_Plotting.html#draw-the-plot-and-legend-for-wwas)<br>  \n",
+    "[9 See Also](https://unidata.github.io/python-awips/examples/generated/Watch_Warning_and_Advisory_Plotting.html#see-also)<br>  \n",
+    "&nbsp;&nbsp;&nbsp;&nbsp;[9.1 Related Notebooks](https://unidata.github.io/python-awips/examples/generated/Watch_Warning_and_Advisory_Plotting.html#related-notebooks)<br>   \n",
+    "&nbsp;&nbsp;&nbsp;&nbsp;[9.2 Additional Documentation](https://unidata.github.io/python-awips/examples/generated/Watch_Warning_and_Advisory_Plotting.html#additional-documentation)<br>   "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1 Imports\n",
+    "\n",
+    "The imports below are used throughout the notebook.  The python-awips and allow us to connect to an EDEX server, use the warning lookup dictionary, and define a TimeRange.  The additional imports are for data manipulation and visualization. "
```

Comment:
Something is up with this second sentence - maybe you started editing it but didn't finish?